setup.cfg

```diff
@@ -0,0 +1,31 @@
+[metadata]
+name = sktm
+description = "Sonic Kernel Testing Manager"
+long_description = file: README.md
+version = 1
+author = Red Hat, Inc.
+license = GPLv2+
+
+[options]
+# Automatically find all files beneath the sktm directory and include them.
+packages = find:
+# Parse the MANIFEST.in file and include those files, too.
+include_package_data = True
+
+install_requires = flake8
+                   dateutils
+                   enum34
+                   jenkinsapi
+                   junit_xml
+                   requests
+                   pylint
+                   mock
+
+[options.entry_points]
+console_scripts =
+    sktm = sktm.executable:main
```

Comment:
This change is done in two steps: in one commit, the `sktm:main` part is added, and then it's changed to include the `executable`. Can you do both in a single step?